routes/discord.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const authorizeBot = require("../middlewares/authorizeBot");
+const validator = require("../middlewares/validators/discord");
+const discord = require("../controllers/discord");
+
+router.put("/", validator.discordData, authorizeBot, discord.addDiscordData);
```

Comment:
There isn't any major reason for using `PUT` here but I have used it here as it is idempotent and duplicate resources won't be allowed and if we encounter any same resource it will update that the existing one.
So, we require this here as when the user initiates `/link` command it may happen that the bot server doesn't respond and user retries with same resource so in that case it will be helpful.

But `POST` will also work fine here as we will be handling the `Not responding` in Bot where it is meant to be used.

Reply:
I have changed it to `POST`